### PR TITLE
Update 00000005.txt

### PR DESCRIPTION
**Pull request by willf**

### CODE REVIEW

File: 1957/CORRECTED/00000005.txt [hunk 1]
```diff
@@ -1,14 +1,14 @@
-ANNUAL SACKED HARP SINGINGS
-3
 Fourth—Rock Creek Church, 5 Miles N. E. of Double Springs.
-Fourth—Sullins School House, Marion County, on Hackleburg and Hamil¬
+Fourth—Sullins School House, Marion County, on Hackleburg and Hamil-
 ton Road.
 Fourth, and Saturday before—Courthouse at Guntersville.
+
```
Comment (willf) [inline 1957/CORRECTED/00000005.txt:5]: Are you sure we want to add spaces? @jpkarlsberg what do you think

File: 1957/CORRECTED/00000005.txt [hunk 2]
```diff
@@ -1,14 +1,14 @@
-ANNUAL SACKED HARP SINGINGS
-3
 Fourth—Rock Creek Church, 5 Miles N. E. of Double Springs.
-Fourth—Sullins School House, Marion County, on Hackleburg and Hamil¬
+Fourth—Sullins School House, Marion County, on Hackleburg and Hamil-
 ton Road.
 Fourth, and Saturday before—Courthouse at Guntersville.
+
 June Sundays
+
```
Comment (willf) [inline 1957/CORRECTED/00000005.txt]:
```suggestion
```

File: 1957/CORRECTED/00000005.txt [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
 ton Road.
 Fourth, and Saturday before—Courthouse at Guntersville.
+
 June Sundays
+
 First—Shady Grove Church, 29 Miles West of Cullman, 3 Miles South of
 Highway 69.
 First—Fellowship Church, 8 Miles West of Cullman.
-First—Pine Tucky Church, 5 Miles South of Double Springs, % Mile East
+First—Pine Tucky Church, 5 Miles South of Double Springs, 3/4 Mile East
```
Comment (willf) [inline 1957/CORRECTED/00000005.txt]:
```suggestion
First—Pine Tucky Church, 5 Miles South of Double Springs, ¾ Mile East
```

File: 1957/CORRECTED/00000005.txt (hunk 1 above)
Comment (jpkarlsberg) [reply 1957/CORRECTED/00000005.txt:5]:
```suggestion
```

**Pull request closed without merging**